resources/definitions/os_discovery/pandacom.yaml

```diff
@@ -0,0 +1,438 @@
+mib: PanDacom-MIB:SPEED-MUX-200G-MIB:SPEED-MULTILINE-MIB:SPEED-AMP-MIB:SPEEDSINGLELINE-XFP-MIB:SPEED-DUALLINE-FC:SPEEDCARRIER-MIB
+modules:
+    os:
+        serial: SPEEDCARRIER-MIB::nmCarrierSerialNumber.0
+        serial_replace: 
+            - [ '/unknown/', '']
+        hardware: SPEEDCARRIER-MIB::nmCarrierType.0
+        hardware_replace:
+            - ['/carrier5UHighPower/', 'Carrier 5U HighPower']
+            - ['/carrier5U/', 'Carrier 5U']
+            - ['/carrier45U/', 'Carrier 4.5U']
+            - ['/carrier1U/', 'Carrier 1U']
+            - ['/carrier4U/', 'Carrier 4U']
+        location: 'SPEEDCARRIER-MIB::nmAgentConfigSnmpSysLocation.0'
+
+    sensors:
+        additional_oids:
+            data:
+                -
+                    oid:
+                        - SPEED-MUX-200G-MIB::speedMux200gLinePortXCVThresholdTable
+                        - SPEED-MUX-200G-MIB::speedMux200gClientPortXCVThresholdTable
+                        - SPEED-MULTILINE-MIB::speedMultilinePortXCVThresholdTable
+
+        state:
+            data:
+                -
+                    oid: nmAAlarmState
```

Comment:
oid and value should have `MIB-NAME::` before them

Reply:
fixed